openpype/hosts/maya/api/pipeline.py

```diff
@@ -681,6 +686,116 @@ def before_workfile_save(event):
         create_workspace_mel(workdir_path, project_name)
 
 
+def prompt_warning(message, show_cancel=False):
+    """Show feedback to user.
+
+    Returns:
+        bool
+    """
+
+    accept = QtWidgets.QMessageBox.Ok
+    if show_cancel:
+        buttons = accept | QtWidgets.QMessageBox.Cancel
+    else:
+        buttons = accept
+
+    state = QtWidgets.QMessageBox.warning(
+        None,
+        "",
+        message,
+        buttons=buttons,
```

Comment:
Not sure if fond of ternary, but this could be like so:
```py
buttons = accept | QtWidgets.QMessageBox.Cancel if show_cancel else accept
```

Reply:
That is one mighty long line :)

Got a suggestion to break that up for line limit?

Reply:
```python
buttons = (
    QtWidgets.QMessageBox.Ok | QtWidgets.QMessageBox.Cancel 
    if show_cancel else QtWidgets.QMessageBox.Ok
)
```
Like that?

Reply:
@Minkiu does BigRoy suggestion work for you?